token_whitelist.txt

```diff
@@ -0,0 +1,22 @@
+...
+'''
+(!)
+[...]
+:).
+:))
+***
+(...)
+!!!
+???
+;-)
+(?)
+:-)
+:-(
+(*)
++/-
+:((
+(!!!)
+(%)
+[*]
+:-\
+:-]

```

Comment:
Are these based on the token statistics?

Reply:
Yes, I went through the more frequent ones and choose the ones that seemed most meaningful.